images/docker-compose-motorpigcs2.yml

```diff
@@ -0,0 +1,13 @@
+services:
+  ioc:
+    image: ghcr.io/cnpem/motorpigcs2-epics-ioc:$TAG
+    build:
+      context: ./
+      dockerfile: ../Dockerfile
+      target: no-build
```

Comment:
This IOC should support static linking, but we are using dynamic linking for it unnecessarily, in a way...

I think we could have an option to build these built-in IOCs statically.

Reply:
It was not possible to update to "static-link", a failure is generated in the Dockerfile when copying the RELEASE file.

Reply:
Yeah, I think it won't be possible for now, we need a feature to support it.